WordPress/Classes/Extensions/UIAlertController+Helpers.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+
+extension UIAlertController
+{
+    public func addCancelActionWithTitle(title: String?, handler: ((UIAlertAction) -> Void)?) -> UIAlertAction {
+        return addActionWithTitle(title, style: .Cancel, handler: handler)
+    }
+    
+    public func addDestructiveActionWithTitle(title: String?, handler: ((UIAlertAction) -> Void)?) -> UIAlertAction {
+        return addActionWithTitle(title, style: .Destructive, handler: handler)
+    }
+
+    public func addDefaultActionWithTitle(title: String?, handler: ((UIAlertAction) -> Void)?) -> UIAlertAction {
+        return addActionWithTitle(title, style: .Default, handler: handler)
+    }
+    
+    public func addActionWithTitle(title: String?, style: UIAlertActionStyle, handler: ((UIAlertAction) -> Void)?) -> UIAlertAction {
+        let action = UIAlertAction(title: title, style: style, handler: handler)
+        addAction(action)
+        
+        return action
+    }
+    
+    public func presentFromRootViewController() {
+        // Note:
+        // This method is required because the presenter ViewController must be visible, and we've got several
+        // flows in which the VC that triggers the alert, might not be visible anymore.
+        //
+        guard let rootViewController = UIApplication.sharedApplication().delegate?.window??.rootViewController else {
+            print("Error loading the rootViewController")
+            return
+        }
+        
+        var leafViewController = rootViewController
+        while leafViewController.presentedViewController != nil {
+            leafViewController = leafViewController.presentedViewController!
+        }
```

Comment:
@kurzee i'm certainly not happy with that loop. It _should be_ safe, unless something huge changes.

Personally, i'd love to just remove that snippet altogether. Any suggestions would be more than welcome sir!

By the way, i've just realized that we should be grabbing the *key window* instead. Will submit that fix in another PR!